README.md

```diff
@@ -1,8 +1,16 @@
-# Okta + Android (Java) & Okta-Hosted Login Page Example
+# Redirect login for Android(Kotlin)
```

Comment:
I think the title can be a bit more informative

Reply:
I agree here. I took this suggestion from @mauricesharp-okta, but I agree it could likely be improved. @mauricesharp-okta do you have any thoughts on this?

Reply:
I applied the suggestion. @deepu105 I feel like this still doesn't quite address your comment, but I think it's a good highlight, and more information can be found by reading further. But if you feel strongly, let us know and we can come up with something better.